app/models/category.rb

```diff
@@ -1,6 +1,7 @@
 class Category < ActiveRecord::Base
-  has_many :loan_request_categories
+  has_many :loan_request_categories, dependent: :destroy
   has_many :loan_requests, through: :loan_request_categories
+  default_scope { order('updated_at DESC') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.